app/controllers/application_controller.rb

```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+  add_flash_types :success, :info, :warning, :danger
+  before_action :login_required
+  helper_method :current_user
+
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
```

Comment:
```suggestion
    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

のほうが良さげ。
しっかりメモ化使ってていいっすね〜。